examples/account/README.md

```diff
@@ -58,7 +60,9 @@ commands will detect it and remind you to do so if necessary.
 
 # Apply
 
-In a real-world deployment, you likely want to perform a `terraform apply` first. See [Hashicorp's Best Practices](https://www.terraform.io/docs/cloud/guides/recommended-practices/index.html) documentation for more information.
+In a real-world deployment, you likely want to perform a `terraform apply` first.
```

Comment:
wouldnt you want to `terraform plan` before apply?

Reply:
This PR is just reformatting the code without changing content, at leas for the docs.